### PR TITLE
March 2021 Newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
I went into a bit more detail in upcoming features in this month's newsletter.

### CODE REVIEW

File: content/posts/newsletter-march-2021.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## March advertising stats
+
+
+March was another record month ever for advertising
+although [Februrary]({filename}/posts/newsletter-february-2021.md) was better per day due to being a shorter month.
+
+In March 2021:
+
+* We generated **$23,994.57** for our publishers another record but just barely up under 1% over February.
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:23]:
```suggestion
* We generated **$23,994.57** for our publishers, another record but only 1% over February.
```

File: content/posts/newsletter-march-2021.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  Of this, $10,711.54 was publishers other than our own publisher (Read the Docs).
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:24]:
```suggestion
  Of this, $10,711.54 was to third-party publishers, with the rest to Read the Docs.
```

File: content/posts/newsletter-march-2021.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  These included showing more data, giving more high level overviews of data, and speeding up our existing reports.
+* We started automatically including the publisher name when redirecting to an advertiser's site.
+  The advertising industry could use a bit of transparency
+  and advertisers sometimes don't know where their ads are being shown.
+  Our aim is to be as transparent as possible and give advertisers the ability to make more informed decisions.
+* We made a small tweak to our ad display algorithm when the same person requests multiple ads within a few seconds.
+  We now cache the ad and show the exact same ad and only bill advertisers a single impression for it.
+  For most publishers, this made almost no difference but for some this resulted in fewer ads being shown
+  and therefore less revenue.
+  We believe this is more fair to advertisers and incentivizes publishers to have good ad placements.
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:49]: We should note this in the publisher policy as well, it's been on my list for a while, but needs to be in there.

File: content/posts/newsletter-march-2021.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  which is an important goal and requires constant effort.
+* On our ad server, we made a number of reporting improvements for both publishers and advertisers.
+  These included showing more data, giving more high level overviews of data, and speeding up our existing reports.
+* We started automatically including the publisher name when redirecting to an advertiser's site.
+  The advertising industry could use a bit of transparency
+  and advertisers sometimes don't know where their ads are being shown.
+  Our aim is to be as transparent as possible and give advertisers the ability to make more informed decisions.
+* We made a small tweak to our ad display algorithm when the same person requests multiple ads within a few seconds.
+  We now cache the ad and show the exact same ad and only bill advertisers a single impression for it.
+  For most publishers, this made almost no difference but for some this resulted in fewer ads being shown
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:47]:
```suggestion
  For most publishers, this made almost no difference but for some this resulted in fewer ad views being counted
```

File: content/posts/newsletter-march-2021.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  and therefore less revenue.
+  We believe this is more fair to advertisers and incentivizes publishers to have good ad placements.
+* In our infrastructure, we've solved most of our performance/IO problems from the previous month.
+  We are archiving data out of our transactional database on a regular basis after it is processed
+  to help with performance.
+  Longer term, we will probably use a separate reporting database for both performance
+  but for now, we're keeping it simple.
+
+You can always see our latest updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html).
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:56]:
```suggestion
You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html).
```

File: content/posts/newsletter-march-2021.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+* Beginning in April, we're going to expand our advertising logic to take into account an ad's price and performance
+  when choosing which ad to show.
+  One advertiser has agreed to be a flex advertiser with part of their ad buys in Q2
+  which will give them lower prices but not guaranteed inventory and this feature was a necessary step.
+  This helps us ensure that as many ad slots have paid ads for our publishers while letting us balance our inventory.
+  Initially, we expect this will drive a slight increase in revenue to publishers as we prioritize higher priced ads.
+  Longer term, this aligns incentives to have better performing ads for advertisers and better performing placements on publisher sites.
+* We are building improvements into both our ad client and ad server to improve our contextual targeting.
+  We believe that good contextual targeting is one of the keys to our network's success.
```
Comment (ericholscher) [inline content/posts/newsletter-march-2021.md:87]: We should probably have a blog post or page on the site explaining the difference, so we can link to it from places like this. Maybe in the Ad Vision? But feels like we probably want an "explainer" which isn't a great fit there. Maybe just linked from the footer & publisher pages or something? Or maybe even just an FAQ item?

**Review by ericholscher — APPROVED** [review]
Looks good! 

Want to drop a tweet into the Description, which I can post when it ships?

File: content/posts/newsletter-march-2021.md (hunk 6 above)
Comment (davidfischer) [reply content/posts/newsletter-march-2021.md:87]: I think a blog on contextual targeting would be great. Something like this: https://www.adex.network/blog/why-we-use-contextual-targeting/

**Comment by davidfischer** [comment]
Here's a tweet to go with it.

```We shipped a ton of great improvements in March and it was (just barely) a record month. Even more great features are coming this month. Check out our March newsletter.```

**Pull request merged**